manifests/init.pp

```diff
@@ -44,6 +44,8 @@
   $gid          = undef,
   $shell        = '/bin/true',
 
+  $no_websudo   = false,
```

Comment:
Agreed on the confusing parameter, unfortunately this reversed approach has been decided on by Atlassian, see the configuration item "jira.websudo.is.disabled". Reversing their decision by naming this parameter "websudo" may cause confusion as well. Nevertheless I'll update the parameter to a positive approach to controlling this feature.